str_analysis/data/known_pathogenic_variant_catalog__GRCh38__59_loci_with_offtargets.json

```diff
@@ -0,0 +1,2499 @@
+            "Inheritance": "AD",
+            "OMIM": "609637",
+            "NormalMax": 15,
+            "PathogenicMin": 25
+         }
+      ],
+      "OfftargetRegions": [
+         "chr12:65563559-65563902",
+         "chr16:24729510-24729868",
+         "chr16:24776892-24777296",
+         "chr16:56815697-56816082",
+         "chr16:73546509-73546894",
+         "chr17:10995550-10995927",
+         "chr17:51831513-51831885",
+         "chr18:55585990-55586383",
+         "chr19:13207704-13208051",
+         "chr1:21176412-21176763",
+         "chr2:224585004-224585351",
+         "chr2:86914126-86914735",
+         "chr5:166979020-166979397",
+         "chr5:443060-443432",
+         "chr6:104019162-104019520",
+         "chr6:117907024-117907372",
+         "chr6:16327473-16327876",
+         "chr6:170561754-170562172",
+         "chr6:45422595-45422950",
+         "chr7:100694148-100694545",
+         "chr7:105013999-105014347",
+         "chr7:122288035-122288404",
+         "chr9:20414125-20414554",
+         "chrX:147911890-147912265",
+         "chrX:150983240-150983612",
+         "chrX:19990766-19991128",
+         "chrX:21367060-21367421",
+         "chrX:24364087-24364601",
+         "chrX:67545156-67545569"
+      ]
+   },
+   {
+      "LocusId": "ZIC3",
+      "LocusStructure": "(GCN)*",
+      "VariantType": "RareRepeat",
+      "ReferenceRegion": "chrX:137566826-137566856",
+      "RepeatUnit": "GCN",
+      "InExpansionHunterVariantCatalog": false,
+      "Gene": "ZIC3",
+      "Inheritance": "XR",
+      "GeneRegion": "exon - polyalanine",
+      "Diseases": [
+         {
+            "Symbol": "VACTERLX",
+            "Name": "X-linked VACTERL syndrome",
+            "Inheritance": "XR",
+            "OMIM": "314390",
+            "NormalMax": 10,
+            "PathogenicMin": 12
+         }
+      ],
+      "OfftargetRegions": [
+         "chr12:65563559-65563902",
+         "chr16:24729510-24729868",
+         "chr16:24776892-24777296",
+         "chr16:56815697-56816082",
+         "chr16:73546509-73546894",
+         "chr17:10995550-10995927",
+         "chr17:51831513-51831885",
+         "chr18:55585990-55586383",
+         "chr19:13207704-13208051",
+         "chr1:21176412-21176763",
+         "chr2:224585004-224585351",
+         "chr2:86914126-86914735",
+         "chr5:166979020-166979397",
+         "chr5:443060-443432",
+         "chr6:104019162-104019520",
+         "chr6:117907024-117907372",
+         "chr6:16327473-16327876",
+         "chr6:170561754-170562172",
+         "chr6:45422595-45422950",
+         "chr7:100694148-100694545",
+         "chr7:105013999-105014347",
+         "chr7:122288035-122288404",
+         "chr9:20414125-20414554",
+         "chrX:147911890-147912265",
+         "chrX:150983240-150983612",
+         "chrX:19990766-19991128",
+         "chrX:21367060-21367421",
+         "chrX:24364087-24364601",
+         "chrX:67545156-67545569"
+      ]
+   }
+]
```

Comment:
(also same comment here about the newline)

Reply:
I can't think of a use-case where it's important to have a new line at the end.